app/src/main/java/com/example/lapnguyen/weather_3/api/ServiceGenerator.java

```diff
@@ -14,27 +14,36 @@
     private ServiceGenerator() {
     }
 
-    private static final String BASE_URL =
-            "https://api.darksky.net/";
+    private static final String BASE_URL_WEATHER = "https://api.darksky.net/";
+    private static final String BASE_URL_LOCATION = "http://locationiq.org/";
 
-    private static Retrofit retrofit;
+    private static Retrofit RETROFIT_LOCATION, RETROFIT_WEATHER;
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName